packages/backend-common/config.d.ts

```diff
@@ -72,10 +75,23 @@ export interface Config {
       /** Default database client to use */
       client: 'better-sqlite3' | 'sqlite3' | 'pg';
       /**
-       * Base database connection string or Knex object
+       * Base database connection string or Knex (PG) object
+       * PG object details come from: https://node-postgres.com/apis/client
        * @visibility secret
        */
-      connection: string | object;
+      connection:
+        | string
+        | Partial<{
+            /**
+             * Password that belongs to the Postgres User
+             * @visibility secret
+             */
+            password: string;
+            /**
+             * Other Knex (PG) object settings
```

Comment:
more generic than that - "other connection settings" will do fine - it's not defined by knex actually, and either way it's not PG specific

Reply:
Good call, updated 👍